inc/Nav_Menus/Component.php

```diff
@@ -59,15 +67,55 @@ public function template_tags() : array {
 		);
 	}
 
+	/**
+	 * Set the navs menus slugs and their description.
+	 *
+	 * @return void
+	 */
+	public function set_navs_menus() {
```

Comment:
Same here, better call this `set_nav_menus()`. Also this should probably be private.